.github/workflows/create-xray-test-plan-and-test-execution-e2e-test-newman-test.yml

```diff
@@ -0,0 +1,96 @@
+name: create-xray-test-plan-and-test-execution-e2e-test-newman-test
```

Comment:
not a huge fan of that name, maybe opt for something a little more concise like what Kevin suggested